backend/config_test.go

```diff
@@ -34,7 +34,11 @@ func TestConfigStoreInternalFetch(t *testing.T) {
 					Name:         "Foo-VPC",
 					TemplateName: "VPC",
 					Capabilities: "CAPABILITIES_IAM",
-					Parameters:   map[string]interface{}{"Name": "ProductionVPC"},
+					Parameters: map[string]interface{}{
+						"Bar":     "123abc",
+						"Name":    "ProductionVPC",
+						"VpcCIDR": "10.21.0.0/16",
```

Comment:
@eyeamera does this look right? I'm not sure if this one was passing before or not, but having these inherit from the top-level config is what I would expect...